assembl/models/auth.py

```diff
@@ -946,6 +946,11 @@ def password_p(self, password):
             # set the new password
             self.password = hash_password(password)
 
+    def anonymous_username(self):
```

Comment:
Not critical, but the pythonic way is to finish this method by adding `return None`. By adding this, the function is explicit that it will always return something.